src/app/home/home.component.html

```diff
@@ -0,0 +1 @@
+<p>home works!</p>
```

Comment:
questo va cambiato in qualcosa di più adatto o va aggiunto un commento fixme

Reply:
Sì certamente, è così perché abbiamo fatto la pr prima di finire con i component

Reply:
dato che quando mergiamo use-angular* andrà a finire in baseline, lasciamolo così ma con un FIXME che ci ricordiamo

Reply:
beh tecnicamente è già stato fixato nel branch home-page dove abbiamo fatto la pagina home però si nel branch attuale ha quello che scrive angular di default

Reply:
ah giusto. comunque ho fatto io un committino. una cosa carina sarebbe fare una micropagina "not implemented - lavori in corso" in modo che tutte le pagine di cui inizializziamo il componente ma non implementiamo abbiano una struttura comunqe